server/sonar-server/src/main/java/org/sonar/server/usertoken/ws/RevokeAction.java

```diff
@@ -74,13 +73,37 @@ private void doHandle(RevokeWsRequest request) {
     }
   }
 
-  private RevokeWsRequest toRevokeWsRequest(Request request) {
-    RevokeWsRequest revokeWsRequest = new RevokeWsRequest()
+  private RevokeRequest toRevokeRequest(Request request) {
+    RevokeRequest RevokeRequest = new RevokeRequest()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00117)